cmd/scale.go

```diff
@@ -255,16 +288,35 @@ func (sc *scaleCmd) run(cmd *cobra.Command, args []string) error {
 		currentNodeCount = len(indexes)
 
 		if currentNodeCount == sc.newDesiredAgentCount {
-			log.Info("Cluster is currently at the desired agent count.")
+			sc.printScaleTargetEqualsExisting(currentNodeCount)
 			return nil
 		}
 		highestUsedIndex = indexes[len(indexes)-1]
 
-		// Scale down Scenario
+		// VMAS Scale in Scenario
 		if currentNodeCount > sc.newDesiredAgentCount {
-			if sc.masterFQDN == "" {
+			if sc.apiserverURL == "" {
 				cmd.Usage()
-				return errors.New("master-FQDN is required to scale down a kubernetes cluster's agent pool")
+				return errors.New("--apiserver (or the deprecated --master-FQDN) is required to scale down a kubernetes cluster's agent pool")
+			}
+
+			if sc.nodes != nil {
+				if len(sc.nodes) > 1 {
```

Comment:
```
if len(sc.nodes) == 1 {
    sc.logger.Infof("There is 1 node in pool %s before scaling down to %d:\n", sc.agentPoolToScale, sc.newDesiredAgentCount)
} else {
    sc.logger.Infof("There are %d nodes in pool %s before scaling down to %d:\n", len(sc.nodes), sc.agentPoolToScale, sc.newDesiredAgentCount)
}
```

Reply:
that way it also handles the "0 node pools" case :) if we hypothetically allow 0 nodes in the future